packages/odyssey-react/src/components/Infobox/Infobox.tsx

```diff
@@ -81,20 +81,14 @@ export const Infobox = withTheme(
   theme,
   styles
 )(
-  forwardRef<HTMLElement, InfoboxProps>((props, ref) => {
+  forwardRef<HTMLDivElement, InfoboxProps>((props, ref) => {
     const { content, actions, heading, variant = "info", ...rest } = props;
 
     const classNames = useCx(styles.root, styles[`${variant}Variant`]);
     const omitProps = useOmit(rest);
 
     return (
-      <Box
-        as="aside"
-        {...omitProps}
-        ref={ref}
-        className={classNames}
-        role="status"
-      >
+      <Box {...omitProps} ref={ref} className={classNames} role="status">
```

Comment:
This is tight coupling. Is `Box` a `div` by default?

Can we be explicit and put `as="div"` to be explicit?

Reply:
`Box` is a `div` by default, yup. Will update! This is what we do elsewhere, but I'd also prefer to be explicit here.